src/main/java/org/tron/core/db/AbstractRevokingStore.java

```diff
@@ -313,12 +313,12 @@ public synchronized void shutdown() {
     System.err.println("******** before revokingDb size:" + size());
     try {
       disable();
-      boolean exit = false;
-      while (!exit) {
+
+      while (true) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Add an end condition to this loop. ([squid:S2189](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2189))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1719&issues=AWcID3GxkEeD8QRkGg6C&open=AWcID3GxkEeD8QRkGg6C)

[ISSUE_KEY]: AWcID3GxkEeD8QRkGg6C